### PR TITLE
News updates

### PR DESCRIPTION
**Pull request by dialexo**
News updates pulled from @GonxheIdrizi (should refork this repo to do proper PR)

### CODE REVIEW

File: src/components/news/content/11/body.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Game* (Game Star) vise à rassembler les actrices et acteurs du jeu vidéo sur le campus de l'UNIL/EPFL. Ses activités sont variées, mais cette commission a notamment participé à l'organisation et organisé de beaux événements, tels que l'eSpace Arlaud lors des Jeux Olympiques de la Jeunesse, Games on Campus, ou encore Game* Speedrun Fest. Une autre de ses missions est d'informer les étudiantes et étudiants sur les possibilités liées au jeu vidéo sur le campus, tant au niveau associatif qu'académique - car des cours sur le jeu vidéo existent !
```
Comment (Battleman) [inline src/components/news/content/11/body.md:1]: Does the stars (`*`) need to be escaped? In GH it render as italic, will it in the website too?

File: src/components/news/content/12/body.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+Il s'agit d'un événement qui se déroule durant le mois de Mars, au bar Satellite de l'EPFL. L'essentiel du travail porte sur prendre contact avec les partenaires au premier semestre, puis préparer l'événement durant le mois de Février-Mars, avec notamment la création d'un thème, la préparation de la communication, et l'organisation de la nourriture et des lots. Il s'agit d'une belle première expérience dans l'associatif, car l'événement tourne correctement depuis trois ans maintenant, et y rencontre des gens formidables, dédiés à la création, dans une ambiance qui change un peu des hackathons traditionnels, orientés "compétition".
```
Comment (Battleman) [inline src/components/news/content/12/body.md:3]: capital M to Mars? French doesn't have capitalized months.. (same to Février)

File: src/components/news/content/12/body.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+
+Intéressé.e ? Envoie un email à orbitalgamestar@groupes.epfl.ch.
```
Comment (Battleman) [inline src/components/news/content/12/body.md:5]: wrong email (orbitalgamejam@groupes.epfl.ch)

File: src/components/news/content/11/body.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+
+Le comité se renouvelle cette année. Game* cherche des gens motivés pour, au choix, monter des événements, ou bien informer le public. La Commission est désormais gentiment lancée, et elle a des fonds assurés pour ses prochaines activités. Au niveau de ses projets, elle souhaite réorganiser spn fonctionnement interne, lancer Games on Campus, et réaliser des ateliers et des conférences en quantité !
+
+Intéressé.e ? Envoie un email à videogames@groupes.epfl.ch
+
+Vous pouvez également suivre les activitées de cette commission sur leur [page Facebook](https://www.facebook.com/CLICGameStar/?eid=ARBJWZi6qZkTlcIiT1-feLzJEoVJUhmR-LsdbB0JmWuZqX8it2BL7bb4ZaTTHYtUrv6exGLNMznAYHVC), leur [canal Telegram](https://t.me/CLICGameStar/) ou encore leur [compte Twitter](https://twitter.com/CLICGameStar).
```
Comment (Battleman) [inline src/components/news/content/11/body.md:7]: Please clean Facebook URL (remove `?eid=...`)

File: src/components/news/content/11/body.md (hunk 1 above)
Comment (dialexo) [reply src/components/news/content/11/body.md:1]: No, the spec requires that there are no spaces between the opening `*` and the first word of the italic bit. GFM is more aggressive in general to allow faster typing, but I've tested the change locally.

**Comment by dialexo** [comment]
@Battleman requested changes were applied (except first, see thread). Dependencies were also brought up-to-date as `npm audit` revealed several new security issues.

**Pull request merged**